src/Views/TryInstallView.vala

```diff
@@ -90,6 +91,8 @@ public class TryInstallView : AbstractInstallerView {
 
         var demo_button = new Gtk.Button.with_label (_("Try Demo Mode"));
 
+        var custom_button = new Gtk.Button.with_label (_("Custom Partitioning"));
```

Comment:
![wtaf](https://user-images.githubusercontent.com/611168/38264683-fd60d700-3730-11e8-8796-1736b9068c2e.png)

The "Custom Partitioning" button here should read `Customize Partitions…` (with the ellipsis since it takes the user elsewhere without performing an action on the current view) and be in [the incidental button position](https://elementary.io/docs/human-interface-guidelines#dialogs) on the DiskView.